### DISCUSSION TITLE
Overriding callback doesnt work

### DISCUSSION THREAD

**Discussion by Coder2195** [nextauthjs/next-auth#5335] [Help]
### Question 💬

![image](https://user-images.githubusercontent.com/63069499/189507632-e7189fe1-a952-4be6-8f62-b4df94caecf0.png)
This code should expose the session with the user profile.
The console.log works fine cuz it exists
![image](https://user-images.githubusercontent.com/63069499/189507641-1f6c29b0-18d2-4689-8c48-227cd984b2bc.png)
But getting the session on the client side doesnt work
This is all it exposes when I overrode the function to add user profile into the session
![image](https://user-images.githubusercontent.com/63069499/189507652-e6babe9a-5d5a-40a7-8ea8-da590497124a.png)

### How to reproduce ☕️

Try to add the user profile into the session callback, it fails miserably on the client side as it doesnt return the user profile

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by ThangHuuVu** [comment]
Are you using the JWT strategy?
try forwarding the user object in the jwt() callback as well

**Reply by ArkShocer** [reply]
@ThangHuuVu Same goes for me besides the user profile is returned correct in the jwt call but undefined in the session call. Any idea on how to send it to the client?

```js
  callbacks: {
    async jwt({ token, account, user }) {
      // Persist the OAuth access_token to the token right after signin
      console.log("JWT: User log")
      console.log(user) // <--- Correct user profile response
      if (account) {
        token.accessToken = account.access_token;
      }
      return {...token, ...user}
    },
    async session({ session, token, user }) {
      // Send properties to the client, like an access_token from a provider.
      console.log("Session: User Log")
      console.log(user) // <---  undefined
      session.accessToken = token.accessToken;
      return {...session, ...user};
    },
  },
```
